.github/workflows/tests.yaml

```diff
@@ -0,0 +1,22 @@
+name: Tests
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '17'
+          cache: 'npm'
+      - run: npm install
+      - name: Validate campaign.js
+        run: npm run validate
+      - run: npx prettier --check .
```

Comment:
I usually create package.json scripts `prettier:check` and `prettier:fix` which hide away prettier parameters. Of course they are pretty simple in this case, but they vary from project to project and remembering them is pain. It is nice to have one command for checking and one for fixing all of the found issues, and that same check command is used in CI.

Reply:
:+1: done